src/Microsoft.ML.Transforms/Text/TextTransform.cs

```diff
@@ -34,8 +34,8 @@ namespace Microsoft.ML.Runtime.Data
     // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
     // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
     // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextTransform"]/*' />
-    public sealed class TextTransform : IEstimator<ITransformer>
+    /// <include file='doc.xml' path='doc/members/member[@name="FeaturizeTextEstimator"]/*' />
+    public sealed class FeaturizeTextEstimator : IEstimator<ITransformer>
```

Comment:
>FeaturizeTextEstimator [](start = 24, length = 22)

@Zruty0 @ivanizo4ka for some transforms we have a separate estimator class, where we instantiate the transform, rather than renaming. Do I need to do that for the ones that don't follow that paradigm, like Text, and Concat, or this is fine?

Reply:
I'm not sure I follow...

---
In reply to: [226817833](https://github.com/dotnet/machinelearning/pull/1328#discussion_r226817833) [](ancestors = 226817833)

Reply:
Take a look at concat; we have a ConcatEstimator and a ConcatTransform. Is this what we want to follow. 

---
In reply to: [227047559](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227047559) [](ancestors = 227047559,226817833)

Reply:
Yes.

---
In reply to: [227058884](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227058884) [](ancestors = 227058884,227047559,226817833)